deploy/requirements.yml

```diff
@@ -41,4 +41,4 @@
 
 - name: opencraft
   src: https://github.com/open-craft/ansible-opencraft
-  version: 'origin/smarnach/opencraft'
+  version: a3e9cde2968e9de94cb9e908244d05654def8dcd
```

Comment:
@e-kolpakov I'm fine adding git tags for this purpose. I'll do this later today.